### DISCUSSION TITLE
Orientation of normal vectors for embedded boundaries

### DISCUSSION THREAD

**Discussion by maikel** [AMReX-Codes/amrex#1819] [Q&A] [answered]
Do normal vectors on the embedded boundary point into the body or into the fluid?

Note: For the implicit function that defines the geometry we read in the documentation

> Define an implicit function of position which describes the surface of the embedded object. Specifically, the function class must have a public member function that takes a position and returns a negative value if that position is inside the fluid, a positive value in the body, and identically zero at the embedded boundary.

**Comment by WeiqunZhang** [comment, marked as answer]
They point into the body, which is consistent with the sign of the implicit function.
